purchases/src/test/java/com/revenuecat/purchases/IdentityManagerTests.kt

```diff
@@ -184,10 +188,39 @@ class IdentityManagerTests {
         assertCorrectlyIdentified("cesar")
     }
 
+    @Test
+    fun `when configuring with a specific user, subscriber attributes are migrated`() {
+        mockCleanCaches()
+        identityManager.configure("cesar")
+        verify {
+            mockDeviceCache.migrateSubscriberAttributesIfNeeded()
+        }
+    }
+
+    @Test
+    fun `when configuring with an anonymous user, subscriber attributes are migrated`() {
+        mockCleanCaches()
+        identityManager.configure(null)
+        assertThat(cachedAppUserIDSlot.captured).isNotNull()
+        verify {
+            mockDeviceCache.migrateSubscriberAttributesIfNeeded()
+        }
+    }
+
+    @Test
+    fun `when configuring, cache is cleaned up`() {
```

Comment:
should there be another one like this but with anon user?

Reply:
Yes! Good catch!